test/frontdesk_test.rb

```diff
@@ -0,0 +1,161 @@
+require_relative 'test_helper'
+
+describe "FrontDesk class" do
+  before do
+    @range = Hotel::DateRange.new(start_date: "1st Apr 2020", end_date: "3rd Apr 2020")
+    @front_desk = Hotel::FrontDesk.new
+  end
+
+  describe "#initialize" do
+    it "creates an instance of FrontDesk class" do
+      expect(@front_desk).must_be_instance_of Hotel::FrontDesk
+    end
+
+    it "has 20 items in a rooms list" do
+      expect(@front_desk.rooms.length).must_equal 20
+    end
+
+    it "creates a list of Room instances" do
+      expect(@front_desk.rooms.last).must_be_instance_of Hotel::Room
+    end
+
+    it "gives a correct room number for the first instance in the rooms list" do
+      expect(@front_desk.rooms.first.number).must_equal 1
+    end
+
+    it "gives a correct room number for the last instance in the rooms list" do
+      expect(@front_desk.rooms.last.number).must_equal 20
+    end
+
+    it "creates a list of reservations" do
+      expect(@front_desk.reservations).must_be_instance_of Array
+    end
+  end
+
+  describe "#get_bookings" do
+    before do
+      @front_desk.reserve_room(@range)
+    end
+
+    it "gives an instance of Reservation in a list of reservations" do
```

Comment:
Consider adding a test to check that the array of reservations is the length that you expect